README.md

```diff
@@ -8,29 +8,41 @@
 <p>A Text-to-Speech Transformer in TensorFlow 2</p>
 </h2>
 
-Implementation of an autoregressive Transformer based neural network for Text-to-Speech (TTS). <br>
-This repo is based on the following paper:
+Implementation of a non-autoregressive Transformer based neural network for Text-to-Speech (TTS). <br>
+This repo is based on the following papers:
 - [Neural Speech Synthesis with Transformer Network](https://arxiv.org/abs/1809.08895)
+- [FastSpeech: Fast, Robust and Controllable Text to Speech](https://arxiv.org/abs/1905.09263)
 
 Spectrograms produced with LJSpeech and standard data configuration from this repo are compatible with [WaveRNN](https://github.com/fatchord/WaveRNN).
 
+#### Non-Autoregressive
+Being non-autoregressive, this Transformer model is:
+- Robust: No repeats and failed attention modes for challenging sentences.
+- Fast: The generation of a mel spectogram takes about 0.04s on a GeForce RTX 2080.
```

Comment:
Are these numbers correct? Would it make sense to benchmark it as well against the autoregressive Transformer?